LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 1Password

```

Comment:
I'm not sure about this, but I think the copyright data should include 2019 as well as 2024. Even if the licensing is new, the copyright should include the creation and modification datas.